src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -50,16 +50,14 @@ private static CompletionItem CreateWorker(
             builder.AddRange(properties);
 
         if (insertionText != null)
-        {
             builder.Add(KeyValuePairUtil.Create(InsertionTextProperty, insertionText));
-        }
 
         builder.Add(KeyValuePairUtil.Create("ContextPosition", contextPosition.ToString()));
         AddSupportedPlatforms(builder, supportedPlatforms);
         symbolEncoder(symbols, builder);
 
         tags = tags.NullToEmpty();
-        if (symbols.All(symbol => symbol.IsObsolete()) && !tags.Contains(WellKnownTags.Deprecated))
+        if (!tags.Contains(WellKnownTags.Deprecated) && symbols.All(static symbol => symbol.IsObsolete()))
```

Comment:
>!tags.Contains(WellKnownTags.Deprecated) [](http://example.com/codeflow?start=12&length=40)

order switched here as the IsObsolete checks allocate quite a bit. I doubt the caller has this tag set already in tags, but this seemed nicer.